src/app/app.component.ts

```diff
@@ -126,8 +126,57 @@ export class MyApp implements OnInit, AfterViewInit {
       this.handleBackButton();
       this.appRatingService.checkInitialDate();
       this.getUtmParameter();
+      this.checkForCodeUpdates();
+      this.platform.resume.subscribe(() => this.checkForCodeUpdates());
     });
   }
+  checkForCodeUpdates() {
+    this.preferences.putString(PreferenceKey.DEPLOYMENT_KEY,"agojO-OZt4dZlt_pu9r9j2Ipy_jY90dbb065-3633-45a5-9c55-c0405eafaebb").toPromise().then();
```

Comment:
We can put the deployment key in the properties file and access it from there.

Reply:
I think this is a temporary implementation which will be removed eventually in next iteration. This is just to inform notification Handling User Story owner to use this statement as part of notification handling